src/utils.ts

```diff
@@ -37,16 +39,26 @@ export interface IPackageInfo {
 	name: string;
 	version: string;
 	aiKey: string;
+	requiredDotNetCoreSDK: string;
+	projectTemplateNugetId: string;
+	minSupportedPostgreSQLProjectSDK: string;
+	maxSupportedPostgreSQLProjectSDK: string
 }
 
-export function getPackageInfo(packageJson: any): IPackageInfo {
-	if (packageJson) {
+export function getPackageInfo(): IPackageInfo {
+	if (!packageInfo) {
```

Comment:
Hm.. who is setting the ```packageInfo```? I assume this was supposed to implement lazy initialization, for which you should assign ```packageInfo``` in the ```if``` block, rather than returning a new object.